### ISSUE TITLE
last edited still the same as first facility

### ISSUE THREAD

**Issue by samholics** [coronasafe/care_fe#1770]
when checking the last edited in the patient card, its still shows the patient card created facility's or user's name only
even if the patient card came through various facilities.

* logged in as district admin
* selected a patient card
* last edited is showing still the same as first facility who created the patient card ( even if the patient card shifted and recieved by any other facilities) 

![image](https://user-images.githubusercontent.com/75214902/130974287-8897fbde-01ce-4a87-ae1e-5ae0078fc1e5.png)
in the above case the patient is admitted in xyz hospital on 17th aug and they updated the cosultation also but the last edited is the initial facility name, but the time ( 25th aug) is xyz hospital's  updated time. ( in this particular case the patient came from several facilities; phc>dcc> Ab hospital>> xyz hospital)

 so there is a confustion among the facilities that someone else trying to edit the patient card.

**Comment by arpancodes** [comment]
Hey @samholics could you verify if you are talking about the `/shifting/:id` page?
also could you provide us patient details so we can test

**Comment by samholics** [comment]
hey @arpancodes i recreated the issue on staging nd adding the video below so pls go through....
[last edited.zip](https://github.com/coronasafe/care_fe/files/7061818/last.edited.zip)

**Comment by arpancodes** [comment]
Hey @samholics actually that's different 😅, that last edited is for the patient details not for the consultation update, for example I changed the patient details from the "update details" button in the patient page and the last edited was updated to my name.
![image](https://user-images.githubusercontent.com/42417893/131076676-2bb92b2b-2ffc-494c-8f91-c49f64c9bfac.png)

And in the consultation page it is showing you the name of the person who edited the consultation.
so to summarize
Patient page shows you who created that patient, and who edited the details of patient, say changing the dose of vaccination. And it doesn't show who edited the last consultation, that's in the consultation as you pointed out in the video. 
Hope that helps... 
Closing the issue, please comment to reopen, if you have any concerns, or just give a 👍🏻 to this comment.
